client/src/components/RichTextEditor.js

```diff
@@ -100,6 +114,12 @@ const importerConfig = {
     }
 
     return null
+  },
+  htmlToStyle: (nodeName, node, currentStyle) => {
+    if (nodeName === "mark") {
```

Comment:
Shouldn't you also check for `"strike"` (counterpart of `styleToHTML`) or am I missing something?
(Assuming there were good reasons to add mark/strike.)

Reply:
I was also surprised when I implemented it, but _mark_ is a case which is not supported by default by the editor (unlinke _strike_).